GothicVRProject/GothicVR
# Feature/kiss the cook

# Hints
* We need to wait for https://github.com/GothicVRProject/GothicVR/pull/156 being merged and pulled into this branch.

# To test
1. Start game
2. Check if NPCs have different heads and body color
3. Check if Snaf is there (Hint: His head is slightly misplaced, but I will fix this later)


# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene

# Testing
